code/game/machinery/medical_kiosk.dm

```diff
@@ -0,0 +1,85 @@
+/obj/machinery/medical_kiosk
```

Comment:
This file is unchecked in the .dme, which is making Travis fail

Reply:
Just updated the .dme. On an unrelated note, this is my first PR to include it's own file that would be CHECKED by the .dme. neat.